javascriptv3/example_code/mediaconvert/README.md

```diff
@@ -1,12 +1,15 @@
 # TypeScript environment for Amazon Lambda examples
-Environment for AWS SDK for JavaScript (V3) Amazon Elemental MediaConvert examples. For more information, see the [AWS documentation for these examples](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/mediaconvert-examples.html).
+Environment for AWS SDK for JavaScript (v3) Amazon Elemental MediaConvert examples. 
```

Comment:
Amazon Elemental MediaConvert -> AWS Elemental MediaConvert, here and throughout.

Reply:
Done